pkg/rancher-desktop/config/settings.ts

```diff
@@ -505,8 +510,8 @@ function updateSettings(settings: Settings) {
   }
   let loadedVersion = settings.version || 0;
 
-  if (loadedVersion < CURRENT_SETTINGS_VERSION) {
-    for (; loadedVersion < CURRENT_SETTINGS_VERSION; loadedVersion++) {
+  if (loadedVersion <= CURRENT_SETTINGS_VERSION) {
+    for (; loadedVersion <= CURRENT_SETTINGS_VERSION; loadedVersion++) {
```

Comment:
This would be easier if we just bumped it to version 8 instead. Otherwise on the next bump we'll get very confused.